lib/config/generator.js

```diff
@@ -0,0 +1,210 @@
+var path = require('path');
+var Vow = require('vow');
+var vowNode = require('vow-node');
+var Table = require('cli-table');
+var prompt = require('prompt');
+var fs = require('fs');
+
+var Checker = require('../checker');
+var utils = require('../utils');
+
+prompt.message = '';
+prompt.delimiter = '';
+prompt.start();
+
+/**
+ * Script that walks the user through the autoconfig flow
+ * @type {String[]}
+ */
+var prompts = [
+    {
+        name: 'Please choose a preset number:'.green,
+        require: true,
+        pattern: /[0-9]*/
```

Comment:
This question could be deferred, in fact, i would like to do https://github.com/jscs-dev/node-jscs/issues/109 (if that something we're willing to do).

But it seems identification through `jscs-` is our best bet.